### DISCUSSION TITLE
xtls协议失效吗？

### DISCUSSION THREAD

**Discussion by wangmice** [XTLS/Xray-core#584] [Q&A]
昨天开始突然掉线，443端口被封，改成别的端口，比如8443，能上一会，大概1分钟不到，又是被封，而其他协议比如ssh是可正常连接的，说明ip没有被封。是不是被识别了？希望有人测试一下

**Comment by satsukk1** [comment]
Obviously, today is a special day.

**Comment by chanhsiaoyu** [comment]
确实，今天重点针对了443，估计是没识别出来但是把怀疑对象先封了再说？
